### PR TITLE
add relative information about add-module-exports to support ordinary…

### LINKED ISSUE #362: Export "ordinary" browser library, without `default` property
Is there a way to build my project so I can use it like other ordinary libraries (like jQuery, moment.js etc.).

When I build my project and I load the file in the dist folder in an html file via a `<script src="myLib.js">` element I get an object containing a `default` property. E. g. `myLib.default.initMyLib()`.

What I want instead is an object like this: `myLib.initMyLib()`. Is this possible using this boilerplate? If so, how?

### PR DESCRIPTION
**Pull request by gabrielstuff**
… lib, fixes #362
As asked here https://github.com/babel/generator-babel-boilerplate/issues/362
Tell me if  I should put it somewhere else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +180,25 @@ An example .babelrc file with this preset configured can be seen
 [here](https://github.com/jmeas/moolah/blob/ee451a9395b3169378f1df506d3a6142201e5306/.babelrc#L5).
 </details>
 
+<details>
+<summary>
+  **How can I export default without hacking - eg. support ordinary libs?**
```
Comment (jmeas) [inline README.md]: Maybe this can be:

> "How can I export my library without the "default" property?"

File: README.md [hunk 2]
```diff
@@ -180,6 +180,25 @@ An example .babelrc file with this preset configured can be seen
...
+</summary>
+
+As stated here, https://github.com/59naga/babel-plugin-add-module-exports :
```
Comment (jmeas) [inline README.md]: Can you remove the space before this colon?

**Comment by gabrielstuff** [comment]
done. Hope you like it.

**Comment by jmeas** [comment]
Thanks @gabrielstuff ! I'll merge this in once CI passes :)

**Comment by jmeas** [comment]
Well, no idea what's up with Node 5. I'm gonna merge this and fix that in a subsequent PR.

**Pull request merged**